mappings/net/minecraft/client/gui/screen/ingame/ForgingScreen.mapping

```diff
@@ -11,3 +11,7 @@ CLASS net/minecraft/class_4894 net/minecraft/client/gui/screen/ingame/ForgingScr
 		ARG 3 mouseY
 		ARG 4 delta
 	METHOD method_25445 setup ()V
+	METHOD method_48467 drawBackgroundExtras (Lnet/minecraft/class_4587;II)V
```

Comment:
This is specifically for drawing the arrow with a cross through it for when the recipe is invalid